src/objecttypes/setup_configuration/tests/test_site_config.py

```diff
@@ -0,0 +1,90 @@
+from pathlib import Path
+
+from django.contrib.sites.models import Site
+from django.test import TestCase
+
+from django_setup_configuration.exceptions import PrerequisiteFailed
+from django_setup_configuration.test_utils import build_step_config_from_sources
+
+from objecttypes.setup_configuration.steps import SitesConfigurationStep
+
+DIR_FILES = (Path(__file__).parent / "files/sites").resolve()
+
+
+class SitesConfigurationStepTests(TestCase):
+    def test_valid_setup_default(self):
+        sites = Site.objects.order_by("pk")
+        site = sites[0]
+        self.assertEqual(sites.count(), 1)
+        self.assertEqual(site.domain, "example.com")
+        self.assertEqual(site.name, "example.com")
+
+        setup_config = build_step_config_from_sources(
+            SitesConfigurationStep,
+            str(DIR_FILES / "valid_setup.yaml"),
+        )
+        step = SitesConfigurationStep()
+        step.execute(setup_config)
+
+        sites = Site.objects.order_by("pk")
+        self.assertEqual(sites.count(), 3)
+
+        site = sites[1]
```

Comment:
Retrieving the `Site` by it's name here would make the test less error prone (the ordering of the queryset might cause issues) and more explicit. 
```suggestion
        site = Site.objects.get(name="example-1")
```

The assertion on the name below could then be removed. This also applies to the token tests.

Reply:
I'm agree